### ISSUE TITLE
"Alt" versions of functional keys are not working

### ISSUE THREAD

**Issue by Molochnik** [elfmz/far2l#1716]
Maybe its not an issue, just a question, why don't Alt+F... combinations work? F... keys work normally.

**Comment by akruphi** [comment]
Linux desktop environments (GNOME, KDE, etc.) use **Alt+F...** If keyboard shortcuts is set in environment they are **never passed to programs**. You need **release these keyboard shortcuts globally** in settings of your environment.

In https://github.com/akruphi/far2l/wiki#keysshells some tips (in Russian) for release Alt+F1 in GNOME & all keys in **Xfce** (it required additional efforts).

**Comment by Molochnik** [comment]
Thanks! I had no idea that Alt-F7 and Alt-F8 could be used for moving windows.

**Comment by atolismesh** [comment]
> Thanks! I had no idea that Alt-F7 and Alt-F8 could be used for moving windows.

Under Ubuntu you can use gnome-control-center -> Keyboard -> View and customize Shortcuts to disable global shortcuts

**Comment by Molochnik** [comment]
Yes, I got it, thank you! Everything is fine now, It's really Far under Linux, unbelievable.

**Comment by akruphi** [comment]
May be closed. Information added in Help in #1771

**Comment by unxed** [comment]
You can find commands to free key combinations used by DEs here:
https://github.com/unxed/far2ltricks/tree/main/unbindkeys

Can close this now? @elfmz
